### ISSUE TITLE
Stopwatches ticking at same time

### ISSUE THREAD

**Issue by MichaelQuint** [emerleite/node-stopwatch#3]
For my penny auction demo I create a stopwatch array, and make ten timers. Each timer has a button to reset the time to 10, and when they reach zero they reset to 60. No matter how many times I press the button, the stopwatches all tick at the same time. Why is this?

**Comment by emerleite** [comment]
Hi @MichaelQuint. Can you please post some code example?

Thanks!!

**Comment by MichaelQuint** [comment]
// A list of stopwatches for each item
var stopWatches = []

var connection = new sql.Connection(config, function(err)
{
    // Stored Procedure
    var request = new sql.Request(connection);
    //request.input('input_parameter', sql.Int, 10);
    //request.output('output_parameter', sql.VarChar(50));
    request.execute('GetActiveItems', function(err, recordsets, returnValue)
    {
        recordsets[0].forEach(function each(record)
        {
            var tmpRecord = new Item()
            tmpRecord.Id = record.Item_ID
            tmpRecord.Name = record.Title
            testItems.Items.push(tmpRecord)
        })

```
    testItems.Items.forEach(function each(item)
    {
        var stopwatch = new Stopwatch(item.Id, { seconds: 60 })

        stopwatch.on("tick", function(secondsLeft)
        {
            item.TimeLeft = secondsLeft
            SendUpdate(item, false)
        })

        stopwatch.on("end", function()
        {
            stopwatch.stop()
            stopwatch.seconds = 60
            stopwatch.start()
        });

        stopwatch.start()
        stopWatches.push(stopwatch)
    })
});
```

});

**Comment by MichaelQuint** [comment]
You can see an example here

http://mikequint.com/

**Comment by MichaelQuint** [comment]
Where you able to take a look at it?

**Comment by emerleite** [comment]
@MichaelQuint yes. I'm not sure about the problem yet. Can you send a pull request with your fix suggestion?

Cheers
